libraries/cmd/planetary_orbit.cpp

```diff
@@ -28,6 +28,7 @@
 
 #include "cmd/unit_generic.h"
 #include "src/vs_logging.h"
+#include "root_generic/configxml.h"
```

Comment:
I don't know why, but I couldn't compile without explicitly importing configxml.h.

Reply:
probably one of the headers needing it and failing to include it; that'd probably be worth investigating

Reply:
So I've had this issue before. The game would stop compiling after I would make a change to some unrelated file. 
I think the issue here is that some of the includes are nested. 
_a.h -> b.h -> c.h_

When you delete
`#include "b.h" `
from _a.h_, stuff starts breaking down.

Reply:
As a general guide:

- any header file should directly include header files it needs to compile correctly and no more
- any implementation file should directly include header files it needs to compile correctly and no more

We should winnow away at these issues by fixing the includes of the various header/implementation files. It may be a bit painful to do but it'll be worth it in the long run.